#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_confdir.dpatch by  <seneca@localhost.localdomain>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change default directories

@DPATCH@

diff -Naur ledgersmb-1.2.3-new/LedgerSMB/Sysconfig.pm ledgersmb-1.2.3-dirty/LedgerSMB/Sysconfig.pm
--- ledgersmb-1.2.3-new/LedgerSMB/Sysconfig.pm	2007-04-12 18:43:00.000000000 -0400
+++ ledgersmb-1.2.3-dirty/LedgerSMB/Sysconfig.pm	2007-04-16 00:01:23.000000000 -0400
@@ -28,13 +28,13 @@
 );
 
 # if you have latex installed set to 1
-$latex = 1;
+$latex = 0;
 
 # spool directory for batch printing
-$spool = "spool";
+$spool = "/var/lib/ledgersmb/spool";
 
 # path to user configuration files
-$userspath = "users";
+$userspath = "/var/lib/ledgersmb/users";
 
 # templates base directory
 $templates = "templates";
@@ -46,7 +46,7 @@
 $backuppath = $tempdir;
 
 # member file
-$memberfile = "users/members";
+$memberfile = "/var/lib/ledgersmb/users/members";
 
 # location of sendmail
 $sendmail = "/usr/sbin/sendmail -t";
@@ -62,19 +62,18 @@
 $check_max_invoices = 5;
 
 # program to use for file compression
-$gzip = "gzip -S .gz";
+$gzip = "/bin/gzip -S .gz";
 
 # Path to the translation files
-$localepath = 'locale/po';
+$localepath = '/usr/share/ledgersmb/locale/po';
 
 # available printers
 %printer = (
-    Laser => 'lpr -Plaser',
-    Epson => 'lpr -PEpson',
+    lpr => 'lpr',
 );
 
 my %config;
-read_config( 'ledgersmb.conf' => %config ) or die;
+read_config( '/etc/ledgersmb/ledgersmb.conf' => %config ) or die;
 
 # We should clean this code up for 1.3 :-) Chris T.
 
